cmd/build.go

```diff
@@ -157,7 +171,53 @@ func newBuildCmd() *cobra.Command {
 				return err
 			}
 
-			api := apiNew(entry.APIURL, entry.Token)
+			if entry.UUID == "" {
```

Comment:
I think it would be better to declare the following just before this line.
That way, the part from line 212 to line 214 will be unnecessary.
```
uuidStr := entry.UUID
```

Reply:
fixed